cdap-ui/app/hydrator/controllers/list-ctrl.js

```diff
@@ -185,6 +185,7 @@ angular.module(PKG.name + '.feature.hydrator')
                   :
                     (new Date().getTime() / 1000) - latestRun.starting;
                 }
+                latestRun.duration = window.CaskCommon.CDAPHelpers.humanReadableDuration(Math.round(latestRun.duration));
```

Comment:
need to do the same thing in detail view?

Reply:
Its already shown right in detailed view. The list view was not consistent.